moveit_core/collision_detection/src/collision_common.cpp

```diff
@@ -59,7 +59,7 @@ void CollisionResult::print() const
     for (const auto& contact : contacts)
     {
       RCLCPP_DEBUG_STREAM_THROTTLE(LOGGER, clock, LOG_THROTTLE_PERIOD,
-                                   "\t" << contact.first.first << ", " << contact.first.second);
+                                   "\t" << contact.first.first << ',' << ' ' << contact.first.second);
```

Comment:
This seems odd to me, it is two characters so a string literal is fine here.

Reply:
Thank you for your response <3
I'll modify it and change only the strings which contain one character in it